itests/org.openhab.core.config.dispatch.tests/src/main/java/org/eclipse/smarthome/config/dispatch/test/ConfigDispatcherOSGiTest.java

```diff
@@ -47,7 +50,7 @@
 
     private ConfigDispatcher cd;
 
-    private static final String CONFIGURATION_BASE_DIR = "configurations";
+    private static final String CONFIGURATION_BASE_DIR = "../../../../../target/classes/configurations";
```

Comment:
When using this path it runs on my local machine and on our build servers. It does not run from within the Eclipse IDE.

To let it succeed in the IDE I have to change the path to `"../../../../target/classes/configurations"`.

Reply:
I guess we will have to live with this for now?